src/extension.ts

```diff
@@ -33,11 +34,22 @@ const DEFAULT_DETAIL = "TabNine";
 const PROGRESS_KEY = "tabnine.hide.progress";
 
 export function activate(context: vscode.ExtensionContext) {
+  const pasteDisposable = vscode.commands.registerTextEditorCommand(
```

Comment:
why is it here? maybe it should move to  initValidator ?

Reply:
Since we are overriding the behavior of ctr+v, we need to register as soon as possible the default behavior, i.e. regular paste.
If i's capable of validator, it will unsubscribe this command and register a new one for the validator-paste behavior.